canonicalwebteam/blog/wordpress_api.py

```diff
@@ -181,6 +183,20 @@ def get_media(media_id):
     return process_response(response)
 
 
+def get_user_by_username(username):
+    url = "".join([API_URL, "/users?search=", username])
+    response = api_session.get(url)
+
+    if not response.ok:
+        return None
+
+    processed_response = process_response(response)
+    if not processed_response[0]:
```

Comment:
`if not processed_response:` otherwise it will fail since array is empty

Reply:
How do you mean? 

If using the API `https://admin.insights.ubuntu.com/wp-json/wp/v2/users?search=nottrobin` the user object will be at position **0** of the response array. If none is present there I want to go into the else here. 

Am I doing something wrong?

Reply:
ah, true! Updated it, thanks